collect_app/src/main/java/org/odk/collect/android/widgets/items/ItemsWidgetUtils.kt

```diff
@@ -0,0 +1,63 @@
+package org.odk.collect.android.widgets.items
+
+import org.javarosa.core.model.SelectChoice
+import org.javarosa.form.api.FormEntryPrompt
+import org.javarosa.xpath.parser.XPathSyntaxException
+import org.odk.collect.android.R
+import org.odk.collect.android.exception.ExternalDataException
+import org.odk.collect.android.externaldata.ExternalDataUtil
+import org.odk.collect.android.fastexternalitemset.ItemsetDao
+import org.odk.collect.android.fastexternalitemset.ItemsetDbAdapter
+import org.odk.collect.android.fastexternalitemset.XPathParseTool
+import org.odk.collect.android.widgets.QuestionWidget
+import java.io.FileNotFoundException
+
+object ItemsWidgetUtils {
+
+    @JvmStatic
+    fun loadItemsAndHandleErrors(widget: QuestionWidget, prompt: FormEntryPrompt): List<SelectChoice> {
```

Comment:
I extracted this before realizing I only needed to call `FormEntryPrompt#getSelectChoices` for the GeoJSON stuff.